core/primitives-core/src/version.rs

```diff
@@ -176,6 +178,8 @@ impl ProtocolFeature {
             ProtocolFeature::RejectBlocksWithOutdatedProtocolVersions => 132,
             #[cfg(feature = "protocol_feature_simple_nightshade_v2")]
             ProtocolFeature::SimpleNightshadeV2 => 135,
+            #[cfg(feature = "protocol_feature_restrict_tla")]
+            ProtocolFeature::RestrictTla => 139,
```

Comment:
nit: Out of order, probably due to some merging, can you put it after post state root? 

Any reason to increase the version by 3?

Reply:
> nit: Out of order, probably due to some merging, can you put it after post state root?

Done

> Any reason to increase the version by 3?

That is because there was a 138 when this feature was implemented and it is actually hard to change because of the yaml file (a bunch of things need to be regenerated). There is no harm jumping a few versions for nightly so I think it's okay.